prototypes/index.js

```diff
@@ -297,6 +298,71 @@ const classPrompts = {
 
 
 
+// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------
+// ---------------------------------------------------------------------------
+
+
+// DATASET: nationalParks from ./datasets/nationalParks
+
+const nationalParksPrompts = {
+  getParkVisitList() {
+    /// Return an object containing the names of which parks I need to visit
+    // and the ones I have already visited eg:
+    // {
+    //   parksToVist: ["Yellowstone", "Glacier", "Everglades"],
```

Comment:
I didn't realize it was `Vist` in both files -- was that intentional?

Reply:
Nope, I just can't spell visit! :)